src/process/mod.rs

```diff
@@ -14,6 +14,8 @@ mod kill;
 mod membarrier;
 #[cfg(target_os = "linux")]
 mod pidfd;
+#[cfg(linux_raw)]
+mod pidfd_getfd;
```

Comment:
Rustix aims to avoid having functions available on `linux_raw` and not `libc`. `pidfd_getfd` can be supported in the `libc` backend using `libc::syscall`.